projects/step-core/src/lib/modules/dynamic-forms/components/dynamic-field-group-editor/dynamic-field-group-editor.component.ts

```diff
@@ -313,4 +314,37 @@ export class DynamicFieldGroupEditorComponent implements OnChanges, OnDestroy {
       this.form.enable();
     }
   }
+
+  private isDeepEqual(valueA?: DynamicFieldGroupValue, valueB?: DynamicFieldGroupValue): boolean {
```

Comment:
I would call it "isFieldGroupValueEqual" or move it into DynamicFieldGroupValue. Without reading the description I would otherwise assume that a deep function is recursive.